hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/api/routes.json

```diff
@@ -28,5 +28,44 @@
   "/heatmap/readaccess?startDate=*&path=*&entityType=bucket": "/bucketHeatmap",
   "/heatmap/readaccess?startDate=*&path=*&entityType=key": "/keyHeatmap",
   "/heatmap/readaccess?startDate=*&path=*&entityType=volume": "/heatmap",
-  "/features/disabledFeatures": "/disabledFeatures"
+  "/features/disabledFeatures": "/disabledFeatures",
+  "/keys/open?limit=0": "/keysOpenSummary",
+  "/keys/deletePending?limit=1": "/keysdeletePendingSummary",
```

Comment:
do limit=0 or limit=1 as used in URI as fixed is correct? Or this is only test URI ?

Reply:
@sumitagrawl  @ArafatKhan2198 
For delete pending API /keys/deletePending?limit=1" we have limit 1 suhhested by arafat with limit=0 it will fetch large amount of data as we need key summary information so used limit 1.